src/couch_log/src/couch_log_trunc_io.erl

```diff
@@ -352,7 +352,7 @@ map_body(Map, Max, #print_options{depth = Depth}) when Max < 4; Depth =:= 0 ->
         _ -> {"...", 3}
     end;
 map_body(Map, Max, Options) ->
-    case maps:to_list(Map) of
+    case lists:sort(maps:to_list(Map)) of
```

Comment:
I think we should update the test instead. 

ping @chewbranca

Reply:
This particular fix is for [these tests](https://github.com/apache/couchdb/blob/main/src/couch_log/src/couch_log_trunc_io.erl#L856-L956), which have not been recently changed.

However, [this test](https://github.com/apache/couchdb/blob/main/src/couch_log/test/eunit/couch_log_formatter_test.erl#L35-L44) is recent and I agree it should be fixed. I've disabled it for now.

Reply:
I re-enabled it with [a workaround](https://github.com/apache/couchdb/pull/4581/commits/bbda103b304332e2736da30cbbcfe1f03b48683a), but still think it should be fixed.

Reply:
Fixed in https://github.com/apache/couchdb/pull/4581/commits/7408ff8dc6d8771ae09b8640fbeeb424be155674